### PR TITLE
QB-2: Modify readme file

### PR DESCRIPTION
**Pull request by mehedihassandev**
modify readme file

### CODE REVIEW

**Review by sun-mit — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,34 +1,40 @@
...
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+This template provides a straightforward setup for building React applications with Vite. It includes support for Hot Module Replacement (HMR) and ESLint for code linting.
 
-Currently, two official plugins are available:
+## Prerequisites
 
-- [@vitejs/plugin-react](https://github.com/vitejs/vite-plugin-react/blob/main/packages/plugin-react/README.md) uses [Babel](https://babeljs.io/) for Fast Refresh
-- [@vitejs/plugin-react-swc](https://github.com/vitejs/vite-plugin-react-swc) uses [SWC](https://swc.rs/) for Fast Refresh
+Before you proceed, make sure you have Node.js and npm (or yarn) installed on your machine.
```
[the remark this answers is not in the document]
Comment (mehedihassandev) [reply README.md:7]: na ata apner project er node ba npm na apner machine e node and npm install kora ace naki seta dekher kotha bolci

[the remark this answers is not in the document]
Comment (sun-mit) [reply README.md:7]: yes. It just reminder. no problem. After clone you should install yarn or npm.

[the remark this answers is not in the document]
Comment (sun-mit) [reply README.md:7]: @mehedihassandev vai you can modify with check version of node or yarn. Like
1.  yarn --version
2. npm --version

noted we must follow yarn

[the remark this answers is not in the document]
Comment (paul-111) [reply README.md:7]: ooo ok ok

**Review by sun-mit — APPROVED** [review]

**Review by paul-111 — APPROVED** [review]

**Pull request merged**